### DISCUSSION TITLE
Regarding Meowix 23.10

### DISCUSSION THREAD

**Discussion by Nekosis** [Meowix-Linux/Meowix-ISO#6] [Announcements]
Hey everyone,

Nekosis from Meowix here. I regret to inform you that Meowix 23.10 will not be releasing at this time.

Due to unforseen circumstances, I have lost access to my computer. Therefore, I find myself unable to build the ISO for this month. I will see if I can get my computer back in time to release an ISO during October. Otherwise, there may unfortunately be no Meowix 23.10.

# So what can I do?

**Please DO NOT use the Meowix 23.9 ISO to install Meowix.** It contains outdated packages.

Instead, you will need to build Meowix yourself. You will need to gain access to an Arch Linux-based OS:

- For **Microsoft Windows** users, install [ArchWSL](https://github.com/yuk7/ArchWSL).
- For **Apple macOS** users, install a virtual machine with a different Arch-based distro such as [EndeavourOS](https://endeavouros.com/).
- For users of **non-Arch based Linux distros**, install Arch Linux via [Distrobox](https://github.com/89luca89/distrobox).

Then, follow the [build and installation steps](https://github.com/Meowix-Linux/Meowix-ISO/#building-the-archiso-profile) as seen in the README.

We at Meowix apologize for any inconvenience this may have caused.

Thank you for choosing Meowix Linux.

**Comment by midnitefox** [comment]
Hope all goes well for you soon with your PC! Also wanted to say I just discovered this OS on Catppuccin's featured projects list and was immediately filled with joy! Amazing work!

**Comment by Nekosis** [comment]
I have access to my laptop again! However, there is a major issue with the ISO (#7) that would prevent installation. Please see if you can help!
